ros2/nightly/nightly/Dockerfile

```diff
@@ -59,21 +60,12 @@ RUN pip3 freeze | grep pytest \
     && python3 -m pytest --version
 
 # install ros2 packages
-ENV ROS_DISTRO foxy
 RUN mkdir -p /opt/ros/$ROS_DISTRO
 ARG ROS2_BINARY_URL=https://ci.ros2.org/view/packaging/job/packaging_linux/lastSuccessfulBuild/artifact/ws/ros2-package-linux-x86_64.tar.bz2
 RUN wget -q $ROS2_BINARY_URL -O - | \
     tar -xj --strip-components=1 -C /opt/ros/$ROS_DISTRO
-
-# Overwrite setup scripts with ones that point to /opt/ros/$ROS_DISTRO
-RUN mkdir -p /tmp/dir/build \
- && cd /tmp/dir \
- && git clone --depth 1 https://github.com/ros2/ros_workspace.git -b latest \
- && cd /tmp/dir/build \
- && COLCON_CURRENT_PREFIX=/opt/ros/$ROS_DISTRO . /opt/ros/$ROS_DISTRO/local_setup.sh \
- && cmake -DCMAKE_INSTALL_PREFIX=/opt/ros/$ROS_DISTRO ../ros_workspace \
- && make install \
- && rm -r /tmp/dir
+RUN sed -i "s|^\(_colcon_prefix_chain_sh_COLCON_CURRENT_PREFIX\s*=\s*\).*$|\1/opt/ros/$ROS_DISTRO|" \
```

Comment:
Could you add back line break and a comment to remind the reader of purpose for this sed? Perhaps a link to an old rel relevant issue, or just a sentence.